README.md

```diff
@@ -121,5 +117,8 @@ To build the application:
 # Using node v4.2 or greater
 $ npm install
 $ npm run build
-$ ./bin/tick --help
+$ ./build/tick --help
 ```
+
+It's recommended to add an alias in your `.bashrc` (`alias tick='/PATH/TO/tickbin/build/tick'`)
```

Comment:
Now that we've got the package installable via npm we aren't wanting to advise on making an alias, we'll leave that up to the develop.